<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>检查连接状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_cover">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录制点击位置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_one_click">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>点击一次（测试）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_click">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>持续点击</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>441</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>按钮初始化，请滑动屏幕</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_one">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单部机器模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_many">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>多部机器模式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_fast">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>快</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_med">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>中(华为)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_slow">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>117</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>慢(华为)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
